backend/src/Book/Infrastructure/FileHandler.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace App\Book\Infrastructure;
+
+use Symfony\Component\Serializer\Encoder\CsvEncoder;
+use Symfony\Component\Serializer\Normalizer\ObjectNormalizer;
+use Symfony\Component\Serializer\Serializer;
+
+class FileHandler
+{
+    protected $serializer;
+    private $delimiter = ';';
+
+    public function __construct() 
+        {
+            $this->serializer = new Serializer([new ObjectNormalizer()], [new CsvEncoder()]); 
+        }
```

Comment:
**question:** come mai qui non avete utilizzato la dependency injection di symfony ?

leggete bene: https://symfony.com/blog/new-in-symfony-3-2-csv-and-yaml-encoders-for-serializer

😄